catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/SegmentHandler.java

```diff
@@ -104,6 +112,29 @@ private void handleTreValues(Metacard metacard, NitfAttribute attribute, Tre tre
   private Attribute populateAttribute(Metacard metacard, String attributeName, Serializable value) {
     Attribute currentAttribute = metacard.getAttribute(attributeName);
 
+    // check if country code
+    if (COUNTRY_CODE_ATTRIBUTES.contains(attributeName)) {
+      List<Serializable> alpha3CountryCodes = new ArrayList<>();
+      String[] tokenizedCountryCodes = value.toString().split(" ");
```

Comment:
❓ Should we account for the possibility of multiple spaces, i.e., `.split(" *")`, or skip empty `countryCode` strings?